api/backup.go

```diff
@@ -211,6 +211,15 @@ func (api *api) RestoreBackup(unlockPassword string, r io.Reader) error {
 
 	go func() {
 		logger.Logger.Info("Backup restored. Shutting down Alby Hub...")
+		api.svc.Shutdown()
+		// ensure no -shm or -wal files exist as they will stop the restore
+		for _, filename := range []string{"nwc.db", "nwc.db-shm", "nwc.db-wal"} {
```

Comment:
This should be `DATABASE_URI` `DATABASE_URI`-shm and `DATABASE_URI`-wal

Reply:
The backup and restore only works with the default directory structure (nwc.db must be within the WORK_DIR)